jdaviz/app.py

```diff
@@ -922,10 +922,12 @@ def get_subsets_from_viewer(self, viewer_reference, data_label=None, subset_type
         return regions
 
     def get_subsets(self, subset_name=None, spectral_only=False,
-                    spatial_only=False, object_only=False, simplify_spectral=True,
-                    use_display_units=False):
+                    region_type='pixel', spatial_only=False, object_only=False,
```

Comment:
I don't think `region_type` is used?
```suggestion
                    spatial_only=False, object_only=False,
```